### DISCUSSION TITLE
How to retrieve the history to use it for the further testing

### DISCUSSION THREAD

**Discussion by rachelhson** [PKU-DAIR/open-box#88] [Q&A] [answered]
Hello Thank you for this great open source to help our optimization process easier!

I am using Bayesian Optimization algorithm in Openbox. 

After the test is completed, if I would like to continue testing from the previous data.
What would be good way to retrieve the history? 

 `history = advisor.get_history()`

How can I use this history for the next test?  Thank you in advance.

**Comment by jhj0411jhj** [comment, marked as answer]
After initializing a new advisor, just set `advisor.history = previous_history`. We may add a new API to retrieve the history more easily.
